Sources/SWBUtil/FSProxy.swift

```diff
@@ -758,7 +758,7 @@ class LocalFS: FSProxy, @unchecked Sendable {
     }
 
     func listExtendedAttributes(_ path: Path) throws -> [String] {
-        #if os(Windows)
+        #if os(Windows) || os(FreeBSD) // FreeBSD blocked on https://github.com/swiftlang/swift/pull/77836
```

Comment:
note to self: might as well add !OpenBSD before merging this as OpenBSD is known to have explicitly removed support for xattrs.